### ISSUE TITLE
Flow Typed generated resources are not detected

### ISSUE THREAD

**Issue by ghost** [gajus/eslint-plugin-flowtype#251]
Seems like flow typed generated content is not detected.
Just install whatever lib: 
e.g.
```
yarn add express
flow-typed install
```
For the code 
```
import express from 'express';

const app = express();
app.get('/', (req: $Request, res: $Response) => res.send('hello world'));
```
I get the following error:
```
6:20  error  '$Request' is not defined   no-undef
6:35  error  '$Response' is not defined  no-undef
```
But flow can typecheck successfully

**Comment by jasonLaster** [comment]
I'm seeing the same thing when I upgrade to `yarn 27.5`  it is fine on version `24.5`

**Comment by gajus** [comment]
No plan to support global types.
